be/src/olap/olap_common.h

```diff
@@ -169,6 +169,7 @@ enum PushType {
     PUSH_NORMAL = 1,
     PUSH_FOR_DELETE = 2,
     PUSH_FOR_LOAD_DELETE = 3,
+    PUSH_NORMAL_V2 = 4,
```

Comment:
Would better add a comment for these four push types.

Reply:
done